creator/ingest_runs/queries/validation_run.py

```diff
@@ -1,18 +1,21 @@
-from graphene import relay
+from graphene import relay, Field, String
 from graphene_django.filter import DjangoFilterConnectionField
 from graphql import GraphQLError
 from django_filters import FilterSet, OrderingFilter
 
 from creator.ingest_runs.nodes import ValidationRunNode
 from creator.ingest_runs.models import ValidationRun
+from creator.ingest_runs.common.model import State
 
 
 class ValidationRunFilter(FilterSet):
     order_by = OrderingFilter(fields=("created_on",))
 
     class Meta:
         model = ValidationRun
-        fields = {"success": ["exact"]}
+        fields = {
```

Comment:
We can add `data_review` here to allow us to look up validation runs for a given data review.